### DISCUSSION TITLE
subprocess support

### DISCUSSION THREAD

**Discussion by NasserMashpery** [TomSchimansky/CustomTkinter#2509] [Q&A]
is customtkinter support subprocess between two files python with including customtkinter ?

**Comment by dipeshSam** [comment]
**Have you tried to use full path?? It might be helpful.**

Regards.

**Comment by dipeshSam** [comment]
You can either use `pathlib` module or `os.path` library.
Here is an updated version of your code provided in the [issue](https://github.com/TomSchimansky/CustomTkinter/issues/2508#issuecomment-2238877693).

``` Python

from customtkinter import *
from subprocess import call
import os

root = CTk()
root.geometry('200x100')
frame = CTkFrame(root)
frame.pack(pady=20, padx=20)

def Open():
    script_path = os.path.join(os.path.dirname(__file__), 'test.py')
    call(["python", script_path])

btn = CTkButton(frame, text='Open File', command=Open)
btn.pack()

root.mainloop()

```

Regards.

**Comment by NasserMashpery** [comment]
not working :(

‫في السبت، 20 يوليو 2024 في 6:54 ص تمت كتابة ما يلي بواسطة ‪Dipesh
Samrawat‬‏ ***@***.***‬‏>:‬

**Comment by dipeshSam** [comment]
You may try the following:
1. Mention if any error or warnings being shown.
2. Check whether your env. and env. path is correctly set or not.
3. Try with different command or script do debug hidden errors.

Regards.
